tools/SourceKit/lib/SwiftLang/CodeCompletionOrganizer.cpp

```diff
@@ -272,7 +272,7 @@ void CodeCompletionOrganizer::preSortCompletions(
                        [](Completion *const *a, Completion *const *b) {
 
      // Sort first by filter name (case-sensitive).
-     if (int primary = (*a)->getName().compare((*b)->getName()))
+     if (int primary = (*a)->getFilterName().compare((*b)->getFilterName()))
```

Comment:
Unrelated to this PR but in `CodeCompletion.cpp:1484` we sort by case insensitive first and only if that’s equal we compare case-sensitively. Shouldn’t we do the same here? Maybe it would be a good opportunity to add a method to `NullTerminatedStringRef` that compares by case-insensitive followed by case-sensitive.